storage/innobase/handler/handler0alter.cc

```diff
@@ -92,6 +92,7 @@ static const alter_table_operations INNOBASE_ALTER_REBUILD
 	| ALTER_STORED_COLUMN_TYPE
 	*/
 	| INNOBASE_ALTER_VERSIONED_REBUILD
+	| ALTER_CONVERT_FIELD_CHARSET
```

Comment:
I don’t think that we need this flag. InnoDB should simply use the various existing flags for adding and dropping indexes.